### PR TITLE
custom expresion for jhpiego-test domain

### PR DESCRIPTION
**Pull request by lwyszomi**
@NoahCarnahan

### CODE REVIEW

File: custom/eqa/expressions.py [hunk 1]
```diff
@@ -79,6 +79,44 @@ def __call__(self, item, context=None):
...
+        forms = FormAccessors(item['domain']).get_forms(xforms_ids)
+        f_forms = [f for f in forms if f.xmlns == self.xmlns]
+        s_forms = sorted(f_forms, key=lambda x: x.received_on)
+
+        if len(s_forms) > 0:
+            latest_form = s_forms[-1]
+        else:
+            latest_form = None
+        path = 'action_plan/%s/action_plan/%s'
+        incorrect_question = latest_form.get_date(path % (self.section, 'incorrect_questions'))
```
Comment (NoahCarnahan) [inline custom/eqa/expressions.py]: Looks like this line should also be inside the `if latest_form:` block

Comment (NoahCarnahan) [inline custom/eqa/expressions.py]: I think you want `get_data()` not `get_date()`?

File: custom/eqa/expressions.py [hunk 2]
```diff
@@ -79,6 +79,44 @@ def __call__(self, item, context=None):
...
+        if latest_form:
+            support = ', '.join(
+                [
+                    item.get_case_property(x.strip()) for x in
+                    latest_form.get_date(path % (self.section, 'copy-1-of-responsible')).split(',')
```
Comment (NoahCarnahan) [inline custom/eqa/expressions.py]: Same `get_data()`/`get_date()` issue

File: custom/eqa/expressions.py [hunk 3]
```diff
@@ -79,6 +79,44 @@ def __call__(self, item, context=None):
...
+                ]
+            )
+            return {
+                'gap': latest_form.get_data('code_to_text/%s' % incorrect_question),
```
Comment (NoahCarnahan) [inline custom/eqa/expressions.py]: We actually want a slightly different value here. `code_to_text/<incorrect_question>` is a label node, and we want the label of that label from the `corehq.apps.app_manager.xform.XForm` model. As you currently have it here, I believe `gap` will always be the XFormInstance value of the label node, which will always be "OK", or something like that. You can get the value like this:
```python
question_list = Application.get(latest_form.app_id).get_questions()
questions = {x['value']: x for x in question_list}
gap = questions['code_to_text/%s' % incorrect_question].label
```
(this snippet of code is untested)

**Review by NoahCarnahan — CHANGES_REQUESTED** [review]
Left a few comments, otherwise looking good.

**Comment by lwyszomi** [comment]
@NoahCarnahan updated

**Review by NoahCarnahan — APPROVED** [review]

**Comment by NoahCarnahan** [comment]
Looks good thanks!

**Pull request merged**